lblod/ember-rdfa-editor-lblod-plugins
# GN-4816: Fix selecting snippet list when pressing on a checkbox

### Overview

Selecting a snippet list was not working correctly when pressing on the checkbox, see video.

PR fixes that


https://github.com/lblod/ember-rdfa-editor-lblod-plugins/assets/769698/46697cf0-84a1-46d9-a123-efe83521a311



##### connected issues and PRs:

Bug introduced with https://binnenland.atlassian.net/browse/GN-4816


### Setup

1. Checkout

### How to test/reproduce

1. `npm run start` 
2. Check if clicking on the checkbox selects the snippet list correctly now

### Challenges/uncertainties

Events smh...


### Checks PR readiness
